src/main/xerus/monstercat/api/Player.kt

```diff
@@ -166,12 +176,38 @@ object Player: FadingHBox(true, targetHeight = 25) {
 		}
 	}
 	
-	private val pauseButton = ToggleButton().id("play-pause").onClick { if(isSelected) player?.pause() else player?.play() }
-	private val stopButton = buttonWithId("stop") { reset() }
-	private val volumeSlider = Slider(0.0, 1.0, Settings.PLAYERVOLUME()).scrollable(0.05).apply {
-		prefWidth = 100.0
-		valueProperty().listen { updateVolume() }
-	}
+	private val pauseButton = ToggleButton().id("play-pause")
```

Comment:
for improved readability I'm considering to always have a newline after the `=` if the declaration spans multiple lines - what do you think?

Reply:
I think it would be nice to have all the initializers aligned, but I'll experiment with it myself later, so don't worry about it now.